src/overlays/actors/ovl_En_Go2/z_en_go2.c

```diff
@@ -1833,7 +1833,7 @@ void EnGo2_BiggoronEyedrops(EnGo2* this, PlayState* play) {
             this->eyeMouthTexState = 2;
             this->unk_20C = 0;
             this->goronState++;
-            func_800F483C(0x28, 5);
+            Audio_SetMainBgmVolume(0x28, 5);
```

Comment:
Late to the discussion, but since the volume is a `u8` would it be beneficial to have it in decimal?

Reply:
Note volumes are 0-127 (0x7F)
afaik this is not something consistent and would need cleanup eventually
(I don't have a preference atm so :shrug: )

Reply:
Yeah, Dragorn nailed it. Currently, volume is both hex and dec all over the repo and totally not consistent. And I've been avoiding touching volume because there's a whole rabbit hole there I've been avoiding (volume vs. volumeSquared vs. velocity debate).

I'm not actually sure which I'd prefer, dec or vol. I think I'm leaning towards dec for all the volume but it'd be worth switching them all at once

Reply:
fwiw I think decimal would be better too, whenever we get to cleaning it up